### PR TITLE
Update man pages for atomic storage commands.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
Man pages update for https://github.com/projectatomic/atomic/pull/915.

Ping @rhatdan

File: docs/atomic-storage.1.md [hunk 1]
```diff
@@ -84,32 +89,39 @@ command will try to first move its data to some other device in the pool.
...
 Note: You must set --rootfs when setting --lvname.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_NAME.
 
 **--rootfs**
-Mountpath where logical volume for container storage would be mounted.
-E.g. --rootfs="/var/lib/containers". This will mount "container-root-lv"
-on "/var/lib/containers".
+Add the mountpath where logical volume for container storage would be mounted to
+/etc/sysconfig/docker-storage-setup. E.g. --rootfs="/var/lib/containers". This
+will mount "container-root-lv" on "/var/lib/containers".
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: s/container-root-lv/CONTAINER_ROOT_LV_NAME/g

File: docs/atomic-storage.1.md [hunk 2]
```diff
@@ -84,32 +89,39 @@ command will try to first move its data to some other device in the pool.
...
 "%" it is assumed value is suitable for "lvcreate -L".
 Note: You must set --lvname and --rootfs when setting --lvsize.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_SIZE.
 
 **--vgroup**
-The name of the volume group for the storage pool.
+Add the name of the volume group for the storage pool to
+/etc/sysconfig/docker-storage-setup.
+E.g atomic storage modify --vgroup=atomicos would add
+VG=atomicos to /etc/sysconfig/docker-storage-setup.
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: All of the examples should be like this.

File: docs/atomic-storage.1.md [hunk 3]
```diff
@@ -84,32 +89,39 @@ command will try to first move its data to some other device in the pool.
...
-Logical volume size for container storage.
+Add logical volume size for container storage to /etc/sysconfig/docker-storage-setup.
 E.g. --lvsize="20%FREE". It defaults to 40% of all free space.
 --lvsize can take values acceptable to "lvcreate -L" as well
 as some values acceptable to "lvcreate -l". If user intends to pass
 values acceptable to "lvcreate -l", then only those values which
 contains "%" in syntax are acceptable.  If value does not contain
 "%" it is assumed value is suitable for "lvcreate -L".
 Note: You must set --lvname and --rootfs when setting --lvsize.
+This would set docker-storage-setup option: CONTAINER_ROOT_LV_SIZE.
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: +E.g atomic storage modify --llvsize=20%FREE would add
 +CONTAINER_ROOT_LV_SIZE=20%FREE to /etc/sysconfig/docker-storage-setup.

**Comment by shishir-a412ed** [comment]
@rhatdan PTAL.

File: docs/atomic-storage.1.md [hunk 4]
```diff
@@ -73,9 +72,14 @@ the directory must be present for the import to happen successfully.
   Print usage statement
 
 **--add-device**
-Add block devices to storage pool. This command will expand your devicemapper
+Add the block devices names to /etc/sysconfig/docker-storage-setup.
+You can then run /usr/bin/docker-storage-setup to add the block devices
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: You must run ...

File: docs/atomic-storage.1.md [hunk 5]
```diff
@@ -84,39 +88,56 @@ command will try to first move its data to some other device in the pool.
...
 
 **--rootfs**
-Mountpath where logical volume for container storage would be mounted.
-E.g. --rootfs="/var/lib/containers". This will mount "container-root-lv"
-on "/var/lib/containers".
+Add the mountpath where logical volume for container storage would be mounted to
+/etc/sysconfig/docker-storage-setup.
+E.g. atomic storage modify --rootfs="/var/lib/containers"
+--lvname="container-root-lv" will add
+`CONTAINER_ROOT_LV_MOUNT_PATH="/var/lib/containers"` to
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: You need to add comtnet about adding container-root-lv also.

File: docs/atomic-storage.1.md [hunk 6]
```diff
@@ -84,39 +88,56 @@ command will try to first move its data to some other device in the pool.
...
-contains "%" in syntax are acceptable.  If value does not contain
-"%" it is assumed value is suitable for "lvcreate -L".
+Add logical volume size for container storage to
+/etc/sysconfig/docker-storage-setup. It defaults to 40% of all free space.
+--lvsize can take values acceptable to "lvcreate -L" as well as some values
+acceptable to "lvcreate -l". If user intends to pass values acceptable to
+"lvcreate -l", then only those values which contains "%" in syntax are acceptable.
+If value does not contain "%" it is assumed value is suitable for "lvcreate -L".
+E.g. atomic storage modify --rootfs="/var/lib/containers" --lvname="container-root-lv"
+--lvsize=20%FREE will add `CONTAINER_ROOT_LV_SIZE=20%FREE` to
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: Add comment about what container-root-lv will be in the config file.

File: docs/atomic-storage.1.md [hunk 7]
```diff
@@ -84,39 +88,56 @@ command will try to first move its data to some other device in the pool.
...
+E.g atomic storage modify --vgroup=atomicos would add
+`VG=atomicos` to /etc/sysconfig/docker-storage-setup.
 
 # reset OPTIONS
 **-h** **--help**
   Print usage statement
 
 **--graph**
-Root of the docker runtime. atomic will search for either /var/lib/docker or /var/lib/docker-latest, if only one exists, atomic will select it as the default.  If both exists or you are running docker with a graph storage at a non default location, you need to pass this flag.
+Root of the docker runtime. atomic will search for either /var/lib/docker or
```
Comment (rhatdan) [inline docs/atomic-storage.1.md]: Can you change docker runtime to container runtime.

**Comment by shishir-a412ed** [comment]
@rhatdan PTAL.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ d06e64b

**Pull request closed without merging**